src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -12,11 +12,15 @@
         <div class="c-ne__content">
             <div :id="entry.id"
                  class="c-ne__text"
-                 :class="{'c-ne__input' : !readOnly }"
+                 tabindex="0"
+                 :class="{ 'c-ne__input' : !readOnly }"
                  :contenteditable="!readOnly"
-                 @blur="updateEntryValue($event, entry.id)"
-                 @focus="updateCurrentEntryValue($event, entry.id)"
-            >{{ entry.text }}</div>
+                 @blur="updateEntryValue($event)"
+                 @keydown.enter.exact.prevent
+                 @keyup.enter.exact.prevent="updateEntryValueOnEnterKeyPress($event)"
```

Comment:
can we name this method according to what it is doing.  By my understanding all it does it force blur. So we can rename is forceBlur